stix_shifter_modules/sentinelone/configuration/config.json

```diff
@@ -0,0 +1,28 @@
+{
+    "connection": {
+        "type": {
+            "displayName": "SentinelOne"
+        },
+        "host": {
+            "type": "text"
```

Comment:
Is there a default host that is generally used? Also, you should include the regex parameter for validation.
`"regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9_:/\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9_:/\\-]*[A-Za-z0-9])$"`

Reply:
No default host name is being used, regex parameter is now included.